.github/workflows/phpstan/autoload.php

```diff
@@ -154,24 +153,3 @@
         }
     }
 }
-
-/*
- * At this time if _PS_VERSION_ is still undefined, this is likely because
- * - we are on a old PrestaShop (below 1.7.0.0),
- * - the shop hasn't been installed yet.
- *
- * In that case, the constant can be set from another one in the installation folder.
- */
-if (!defined('_PS_VERSION_')) {
```

Comment:
FYI never true, always set in https://github.com/PrestaShop/PrestaShop/blob/develop/config/autoload.php#L28 newly always included directly by composer autoload config